packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -49,7 +50,7 @@ export const styles = css`
         background-color: transparent;
         height: 100%;
         width: 100%;
-        border: ${borderWidth} solid transparent;
+        border: 0 solid transparent;
```

Comment:
I think this breaks most states of the outline primary button. Or something else does. Even though you're shortly going to redo the primary states, it would be nice to keep them correct in the meantime. See if just setting `border-width: ${borderWidth}` for those states fixes them.

Reply:
If I understand correctly, you meant that the border in the active state was not showing up, correct? I fixed this issue, but the only way I was able to do so was by adding the box-shadow property to the active state of the outline primary button.

Reply:
Sorry, should have been more specific. I see three states of the primary outline button that are wrong now:
- default state has thin border
![image](https://github.com/ni/nimble/assets/7282195/18df99c3-01e5-4262-a21c-6051894ae121)
- hover state has thin border
![image](https://github.com/ni/nimble/assets/7282195/0221209b-5341-4e8c-9343-077cab6fd6f4)
- keyboard focused state is missing inner ring, and outer border is thin
![image](https://github.com/ni/nimble/assets/7282195/f244a8cf-3df6-4f0d-a690-cc3a1c007701)

If it's not straightforward to restore these, I think you can probably leave them as-is, since you will immediately be restyling the primary button anyway.